deploy/kustomize/driver/pmem-csi.yaml

```diff
@@ -155,18 +153,14 @@ spec:
         imagePullPolicy: Always
         image: quay.io/k8scsi/csi-node-driver-registrar:v1.X.Y
         args: [ "-v=3",
-            "--kubelet-registration-path=/var/lib/kubelet/plugins/pmem-csi.intel.com/csi.sock",
+            "--kubelet-registration-path=/var/lib/pmem-csi.intel.com/csi.sock",
             "--csi-address=/csi/csi.sock" ]
         volumeMounts:
-        - name: plugin-socket-dir
+        - name: pmem-state-dir
           mountPath: /csi
         - name: registration-dir
           mountPath: /registration
```

Comment:
Is `/registration` used anywhere?

Reply:
No, I guess. Will remove part of this change.

Reply:
Done

Reply:
Oops, I was wrong, this registration path is used by node-driver-registrar to keep its listening socket. I will revert this change.